src/pages/studies/SidebarFilters.tsx

```diff
@@ -36,61 +38,76 @@ const keyEnhanceBooleanOnly = (key: string) => {
   }
 };
 
-const onSearch = (search: string) => {
-  updateQueryFilters(
-    history,
-    'search',
-    [
-      {
-        content: {
-          field: 'search',
-          value: [search.toLowerCase()],
-        },
-        op: 'in',
-      },
-    ],
-    'and',
-  );
-};
-
 type StudiesProps = {
   onChange: () => void;
+  filters: ISqonGroupFilter;
 };
 type OwnProps = SidebarData & StudiesProps;
 
-const SidebarFilters = ({ studiesResults, studiesMappingResults, onChange }: OwnProps) => {
-  const mappingData = studiesMappingResults;
+export interface ItemProps {
+  label: React.ReactElement;
+  value: string;
+}
+
+const sqon = {
+  content: [],
+  op: 'and',
+};
+
+const SidebarFilters = ({ studiesResults, studiesMappingResults, onChange, filters }: OwnProps) => {
   const data = studiesResults;
+  const options: ItemProps[] = [];
 
-  if (mappingData?.loadingMapping || !mappingData?.extendedMapping || data.loading || !data.data) {
-    return null;
+  let allStudies = useGetStudiesSearch({
+    sqon: sqon,
+    first: 10,
+    offset: 0,
+  });
+
+  if (allStudies && allStudies.data) {
+    allStudies.data.hits.edges.forEach((n) =>
```

Comment:
I would go for map vs foreach but it s up to you it s only a matter of personal taste, but it removes the need for let identifier.